web-client/public/js/container.js

```diff
@@ -4,23 +4,42 @@ $(function () {
   var enableScroll = $("#enableScroll");
   var pageWidth = $(window).width();
 
+  var WIDTH_OFFSET = 250;
+  var HEIGHT_OFFSET = 53;
+
+  var windowWidth = $(window).width() - WIDTH_OFFSET;
+  var windowHeight = $(window).height() - HEIGHT_OFFSET;
+
   if (pageWidth < 1500) {
       $('#boundBoxS').prop('checked', true);
-      $(".containerDefault").attr("class", "grnsight-container containerS");
+      container.addClass("containerS");
   } else if (pageWidth > 1500 && pageWidth < 2200) {
```

Comment:
`1500` and `2200` should be given symbolic names so that we know what they represent.